src/PageContainer.js

```diff
@@ -54,7 +109,11 @@ class PageContainer extends Component {
     render() {
         return (
             <div>
-                <PageHeader logo={logo} title={this.props.account.name} />
+                <PageHeader 
+                    notificationClicked={this.notificationClicked.bind(this)} 
+                    logo={logo} 
+                    title={this.props.account.name} 
+                    notificationCount={this.state.notificationCount}/>
```

Comment:
`notificationClicked` and `notificationClicked` would then be something `PageHeader` itself has rather than passed down